packages/vitest-environment-nuxt/src/runtime/render.ts

```diff
@@ -68,6 +67,8 @@ export async function renderSuspended<T>(
     ..._options
   } = options || {}
 
+  const { render : renderFromTestingLibrary } = await importModule<typeof import('@testing-library/vue')>('@testing-library/vue')
```

Comment:
could we just use a dynamic import of `@testing-library/vue` rather than `importModule`?

Reply:
sure :+1: